spring-cloud-kubernetes-fabric8-leader/src/test/java/org/springframework/cloud/kubernetes/fabric8/leader/Fabric8LeadershipControllerTest.java

```diff
@@ -60,4 +74,55 @@ public void shouldGetEmptyLocalLeader() {
 		assertThat(this.fabric8LeadershipController.getLocalLeader().isPresent()).isFalse();
 	}
 
+	@Test
+	void whenNonExistentConfigmapAndCreationNotAllowedStopLeadershipAcquire() {
+
+		class MemoryAppender extends ListAppender<ILoggingEvent> {
```

Comment:
so this is supposed to capture a log and assert it, right? If so, can't we use `OutputCaptureExtension' here? wdyt?

Reply:
I didn't know this possibility existed. I updated the code with OutputCaptureExtension. Thanks.